src/components/PoiTitle.jsx

```diff
@@ -32,10 +35,11 @@ const PoiTitle = ({ poi, withAlternativeName }) => {
   const alternative = (withAlternativeName && name && localName && localName !== name) && localName;
   const subclass = capitalizeFirst(poiSubClass(subClassName));
 
+  // Location / address
   return <div className="poiTitle">
     <h2 className="poiTitle-main u-text--smallTitle">{title || subclass}</h2>
     {alternative && <div className="poiTitle-alternative u-text--subtitle u-italic">
-      {alternative}
+      { filter(address).map(item => <div key={item}>{item}</div>) }
```

Comment:
the same here. Hum maybe this logic could be moved to its own component :)

Reply:
Actually, as we discussed multiple times, there should be an `<Address>` component responsible for all this.

Reply:
How I see it: a "dumb" component rendering a `<div>` (neutral markup, can be wrapped in something else if needed) with at least an `address` prop to pass the object.
Other props that will prove quickly useful include an optional `className` and probably props to change the format (`inline` true/false, `withCountry` true/false, etc.). We don't need to include everything in the beginning, just what is useful right now, but at least we will have introduced consistency and have a well-defined place to do everything related to address format.

Reply:
of course, I'm on it!
I didn't mark this discussion as solved yet, I just wanted to push a first fix this morning to see if the tests behave like I want with the poi.json modification (it does, I know how to fix the tests now :) )